File: Modul 03/04/PRAK304-2410817310020-M. Rizki Dinar.py
arr = [0, 10, 19, 99]
charr = ["Satuan", "Belasan", "Puluhan"]
charDipilih = ''
nilai = int(input())

if nilai == 0:
    print("Nol");
elif nilai == 10:
    print("Puluhan")
elif nilai >= 100:
    print("Anda Menginput Melebihi Limit Bilangan");
else:
    
    for i in range(len(arr) - 1):
        if arr[i] <= nilai <= arr[i + 1]:
            print(charr[i])
            break

File: Modul 03/04/PRAK304-2410817310020-M. Rizki Dinar.c
#include <stdio.h>

int main() {
    int arr[] = {0, 10, 19, 99};
    char *charr[] = {"Satuan", "Belasan", "Puluhan"};
    char *charDipilih = "";
    int nilai, arr_size;
    
    arr_size = (sizeof(arr) / sizeof(arr[0]) - 1);

    scanf("%d", &nilai);

    if (nilai == 0) {
        printf("Nol\n");
    } else if (nilai == 10) {
        printf("Puluhan\n");
    } else if (nilai >= 100) {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    } else {
        for (int i = 0; i < arr_size; i++) {
            if (nilai >= arr[i] && nilai <= arr[i + 1]) {
                printf("%s\n", charr[i]);
                break;
            }
        }
    }

    return 0;
}
